core/src/config/applier/state.cc

```diff
@@ -61,8 +61,9 @@ state::~state() {}
  */
 void state::apply(com::centreon::broker::config::state const& s, bool run_mux) {
   // Sanity checks.
+  logger::instance().apply(s.loggers());
```

Comment:
This line comes from lower. So if you move it here, no need to keep the previous one.